ledger/participant-integration-api/src/test/lib/scala/platform/store/migration/tests/MigrationEtqTests.scala

```diff
@@ -0,0 +1,597 @@
+    fetchTable(
+      Schema126.idFilterCreateStakeholder
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterCreateNonStakeholderInformee
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterNonConsumingInformee
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterConsumingStakeholder
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterConsumingNonStakeholderInformee
+    ) shouldBe empty
+    fetchTable(
+      Schema126.transactionMeta
+    ) shouldBe empty
+  }
+
+  it should "migrate 4 - id filter for event create stakeholders" in {
+    migrateTo(srcMigration)
+    insertMany(
+      Schema125.createEventsFilter -> Seq(
+        createEventsFilter_125(
+          event_sequential_id = 401L,
+          template_id = 411,
+          party_id = 421,
+        )
+      )
+    )
+    migrateTo(dstMigration)
+    fetchTable(
+      Schema126.idFilterCreateStakeholder
+    ) shouldBe Seq(
+      row(
+        "event_sequential_id" -> 401L,
+        "template_id" -> 411,
+        "party_id" -> 421,
+      )
+    )
+  }
+
+  it should "migrate 5 - two transactions" in {
+    migrateTo(srcMigration)
+    insertMany(
+      Schema125.createEvents ->
+        Seq(
+          newCreateEvent_125(
+            transaction_id = "txId501",
+            contract_id = "cId501",
+            event_offset = "eventOffset501",
+            event_sequential_id = 501L,
+            flat_event_witnesses = Vector.empty,
+            tree_event_witnesses = Vector.empty,
+            template_id = 0,
+          ),
+          newCreateEvent_125(
+            transaction_id = "txId502",
+            contract_id = "cId502",
+            event_offset = "eventOffset502",
+            event_sequential_id = 502L,
+            flat_event_witnesses = Vector.empty,
+            tree_event_witnesses = Vector.empty,
+            template_id = 0,
+          ),
+        )
+    )
+    migrateTo(dstMigration)
+    fetchTable(
+      Schema126.idFilterCreateStakeholder
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterCreateNonStakeholderInformee
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterNonConsumingInformee
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterConsumingStakeholder
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterConsumingNonStakeholderInformee
+    ) shouldBe empty
+    fetchTable(
+      Schema126.transactionMeta
+    ) shouldBe Seq(
+      row(
+        "transaction_id" -> "txId501",
+        "event_offset" -> "eventOffset501",
+        "event_sequential_id_first" -> 501L,
+        "event_sequential_id_last" -> 501L,
+      ),
+      row(
+        "transaction_id" -> "txId502",
+        "event_offset" -> "eventOffset502",
+        "event_sequential_id_first" -> 502L,
+        "event_sequential_id_last" -> 502L,
+      ),
+    )
+  }
+
+  private val dbDataTypes = new DbDataTypes(dbType)
+  object Schema125 {
```

Comment:
Why 125 and 126?

Reply:
These were the flyway migration file numbers for PG. They doesn't make much sense so I'll instead use `Src` and `Dst`